corehq/apps/es/tests/test_elastic_sync_multiplexed_command.py

```diff
@@ -204,6 +204,47 @@ def test_delete_index_deletes_the_older_index(self, adapter_patch, input_patch):
         ESSyncUtil().delete_index(HQ_APPS_INDEX_CANONICAL_NAME)
         self.assertFalse(manager.index_exists(app_adapter.index_name))
 
+    def test_remove_residual_indices(self):
```

Comment:
nit: This could be split into multiple tests for the `remove_residual_indices` (one test ensuring active indices are not removed, another ensuring closed indices are removed, another ensuring residual indices are removed). I think it would make each scenario a bit more readable.

Reply:
I have added extra tests for specifically testing closed and known indices. But I wanted to keep a test that is close to real world, with a mix of all type of indices. So I have kept this test too.

Reply:
https://github.com/dimagi/commcare-hq/pull/33491/commits/0edad3d548b2bb2d931a2ffa5163a1da683dc479